src/main/java/seedu/address/logic/Messages.java

```diff
@@ -48,6 +48,11 @@ public static String format(Person person) {
                 .append(person.getAddress().map(Object::toString).orElse("")));
         builder.append("; Tags: ");
         person.getTags().forEach(builder :: append);
+
+        builder.append("; Modules: ");
```

Comment:
Would it be better to change this to `Role` instead? since in user's perspective this represents the role of the contact.